brave/brave-ios
# Fix #1708: Update Rewards Panel Icon to have more contrast in dark mode

## Summary of Changes

This pull request fixes issue #1708 

## Submitter Checklist:


## Test Plan:
- Enable Rewards
- Go into dark mode
- Visit a verified publisher, icon should be easier to see (white checkmark)

## Screenshots:

![Simulator Screen Shot - iPhone 11 Pro - 2019-10-17 at 14 43 19](https://user-images.githubusercontent.com/529104/67037935-7822b780-f0ec-11e9-9290-71965e59715e.png)

## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
